ui/src/components/LandingPage.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
-import { Title } from '@patternfly/react-core';
+import { Title, Grid, GridItem } from '@patternfly/react-core';
 import Header from './common/Header';
 import AppsTableContainer from '../containers/AppsTableContainer';
 import './LandingPage.css';
 
 class LandingPage extends React.Component {
-  constructor () {
-    super();
-    this.state = {};
-  }
-
   render () {
     return (
       <>
         <Header />
-        <Title className="title" size="3xl">Mobile Apps</Title>
-        <AppsTableContainer />
+        <Grid className="container">
```

Comment:
As talked about in person, we should have a common component for all views in the UI. This will include the header and grid layout with margin implementation. This is not split out but will be similar to the following.

```
       <Grid gutter="md" className="container">
          <GridItem span={1} />
          <GridItem span={10}>
            <Grid gutter="sm" className="container">
              <GridItem span={12}>
                <AppOverviewContainer app={this.props.app} className='app-overview-container'/>
              </GridItem>
              <GridItem span={12}>
                <Title className="table-title" size="2xl">
                  Deployed Versions
                </Title>
                <AppVersionsTableContainer className='table-scroll-x'/>
              </GridItem>
            </Grid>
          </GridItem>
          <GridItem span={1} />
        </Grid>```

Reply:
Would you agree that this should be a new PR and task?

Reply:
yes